x-pack/plugins/observability_solution/observability_onboarding/public/assets/auto_detect.sh

```diff
@@ -392,6 +381,10 @@ known_integration_title() {
 
 build_unknown_log_file_patterns() {
   while IFS= read -r log_file_path; do
+    if [ -z "$log_file_path" ]; then
+      continue
+    fi
+
```

Comment:
Not related to the issue, this just removes the wrong `./*.log` custom logs suggestion.